### ISSUE TITLE
Feature request: IO::Socket::INET sendto/recvfrom methods

### ISSUE THREAD

**Issue by Kaiepi** [rakudo/rakudo#2254]
Raw sockets and UDP sockets can be created, but not used in their current state, since they're connectionless and don't use send/recv like TCP sockets do. This makes it so code that needs to use raw sockets, like code dealing with ICMP, needs to be written in C, which is not a trivial task in terms of cross-platform support (*BSD/Mac, Linux, and Windows each handle sending ICMP packets entirely differently).

tl;dr this would allow raw/UDP sockets to be usable in pure Perl 6

**Comment by lizmat** [comment]
ping @Leont
